cypress/e2e/auth.cy.js

```diff
@@ -0,0 +1,104 @@
+const USER_NAME = "CypressJester";
+const USER_EMAIL = "crypressJest@noroff.no";
+const USER_PASSWORD = "12345678";
+const USER_AVATAR =
+  "https://images.pexels.com/photos/1769271/pexels-photo-1769271.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2";
+
+describe("Authorization", () => {
+  beforeEach(() => {
+    cy.wait(500);
+    cy.visit("/");
+  });
+
+  it("Cannot view profile page", () => {
+    cy.visit("/?view=profile");
```

Comment:
Something should happen next, like checking the URL to make sure that its NOT ?view=profile if not logged in